confiture-web-app/src/store/audit.ts

```diff
@@ -57,6 +57,26 @@ export const useAuditStore = defineStore("audit", {
       return response;
     },
 
+    async updateAuditNotes(uniqueId: string, data: { notes?: string }) {
+      // TODO: handle revert to previous notes if api call fails
+      const previousNotes = this.data?.notes;
+
+      try {
+        const response = (await ky
+          .patch(`/api/audiats/${uniqueId}`, {
+            json: data,
+          })
+          .json()) as Audit;
+        this.data = response;
```

Comment:
Pas besoin de récupérer l'audit depuis la requête, puisqu'on sait ce qu'on vient de changer.

```suggestion
        await ky
          .patch(`/api/audiats/${uniqueId}`, {
            json: data,
          })
          .json();
        this.data.notes = data.notes;
```